android/routes/ui/src/main/java/com/trafi/routes/ui/RoutesScreen.kt

```diff
@@ -198,6 +207,39 @@ private fun ColumnScope.LocationSearchBody(
     }
 }
 
+@Composable
+fun RouteSearchTabsList(
```

Comment:
This can be `private` for now as we don't expect SDK users to use this outside of `RoutesScreen()`